pkg/systemd/services/services.jsx

```diff
@@ -926,19 +949,15 @@ const ServicesPageFilters = ({
         onDeleteChip();
     }, [setCurrentTextFilter, onDeleteChip]);
 
+    const onTextFilterChanged = textFilter => {
+        setCurrentTextFilter(textFilter);
+    };
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18115-20230104-081322-a77aca1f-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)